sim/druid/balance/dragonwrath.go

```diff
@@ -0,0 +1,12 @@
+package balance
+
+import (
+	"github.com/wowsims/cata/sim/common/cata"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func init() {
+	cata.CreateDTRClassConfig(proto.Spec_SpecBalanceDruid, 0.072).
+		AddSpell(8921, cata.NewDragonwrathSpellConfig().SupressImpact()). // Moonfire
+		AddSpell(93402, cata.NewDragonwrathSpellConfig().SupressImpact()) // Sunfire
```

Comment:
Please consider having a spell proc chance override. It seems like there are spells with very low proc chance, like Mushrooms and Typhoon - https://www.mmo-champion.com/threads/967211-FAQ-Dragonwrath-Tarecgosa-s-Rest-and-you-a-Balance-Druid

Reply:
Right now I  assume the 'low' proc chance reported is due to the fact that both typhoon and shrooms only get a single proc rool per 'cast' which hits serveral targets, which is to be expected.

Adding a custom proc chance later on would be very easy. I can just add a proc chance overwrite on the `SpellConfig` for the class if testing should confirm this.